packages/react-app/src/contexts/BridgeContext.jsx

```diff
@@ -29,24 +30,30 @@ export const BridgeProvider = ({ children }) => {
   const { ethersProvider, account, providerChainId } = useWeb3Context();
 
   const [receiver, setReceiver] = useState('');
+  const [amountInput, setAmountInput] = useState('');
   const [{ fromToken, toToken }, setTokens] = useState({});
   const [{ fromAmount, toAmount }, setAmounts] = useState({
     fromAmount: BigNumber.from(0),
     toAmount: BigNumber.from(0),
   });
+  const [isTokenInjected, setIsTokenInjected] = useState({
```

Comment:
Hey can you tell me the reason for having `status` as well as `token` here ? 
We could just have a `injectedToken` variable and do a null check instead of using `status` ??

I might be missing an edge case, if so please let me know